### ISSUE TITLE
-Wunused in include/linux/pci-ats.h

### ISSUE THREAD

**Issue by nickdesaulniers** [ClangBuiltLinux/linux#408]
KernelCI reports:
```
    2    include/linux/pci-ats.h:70:12: warning: unused function 'pci_prg_resp_pasid_required' [-Wunused-function]
```

**Comment by nathanchance** [comment]
https://lore.kernel.org/lkml/65d2712b741cc17c72994642458520853e78f6ba.1551295414.git.sathyanarayanan.kuppuswamy@linux.intel.com/

Looks like the author of the offending commit has sent a fix for this.

**Comment by nathanchance** [comment]
Fixed in `next-03012019`: https://git.kernel.org/pub/scm/linux/kernel/git/next/linux-next.git/commit/?id=fff42928ade591969836ff49888d063b829ac888
